### ISSUE TITLE
Is it possible to extend the model on plain xray of bones

### ISSUE THREAD

**Issue by AdityaIyer** [junyuchen245/Unsuprevised_Seg_via_CNN#3]
I was wondering since you mentioned 2d slices of the data if this model can be used on 2d images of plain xray of say bones of hands & feet to segment & find regions of interest. Thanks & lovely work

**Comment by junyuchen245** [comment]
Hi Aditya,

I do think it's possible. But keep in mind that a neural network trained using only the unsupervised loss tends to segment a given image based on the intensity distribution rather than identifying shapes. So if the intensity values vary a lot within your ROIs in the x-ray images, I think it's better to either fine-tune the trained unsupervised network with few GT labels or use the semi-supervised learning (unsupervised loss + weighting par. x supervised loss). 

Thanks,
Junyu
